palantirnet/federated-search-react
# FS-40: Tweak pager and remove style

Ticket: https://palantir.atlassian.net/browse/FS-40

1. Use `button` instead of `link with role="button"`
1. Add beneficial classes
1. Remove superfluous styles

Test:
1. Verify code/style
1. Make sure to `tab` through pager
